### PR TITLE
OSDOCS-3272: Bump to RHEL 8.6

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OSDOCS-3272

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-3272/release_notes/ocp-4-11-release-notes.html#ocp-4-11-rhcos-rhel-8-6-packages

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL? This PR updates the 4.11 RN to mention that RHCOS uses RHEL 8.6. Thank you!

**Review by mike-nguyen — APPROVED** [review]
LGTM

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -81,6 +81,11 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 
 {op-system} now supports `kdump` for the `arm64` architecture in {product-title} 4.11 as a Technology Preview.
 
+[id="ocp-4-11-rhcos-rhel-8-6-packages"]
+==== {op-system} now uses {op-system-base} 8.6
+
+{op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:87]: I think I like what you have better, but I thought I'd leave a small suggestion for that last sentence. I am not sure if there is a way to make it feel less "listy" though. 
```suggestion
{op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features and enhancements, hardware support, and driver updates. 
```

**Review by opayne1 — COMMENTED** [review]
/LGTM up to you on that suggestion. I honestly think I like what you have better.

**Pull request merged**